release_creation/bundle/requirements/v0.0.latest.requirements.txt

```diff
@@ -12,8 +11,9 @@ dbt-databricks @ git+https://github.com/databricks/dbt-databricks.git@1.8.latest
 dbt-trino @ git+https://github.com/starburstdata/dbt-trino.git@master
 dbt-fabric @ git+https://github.com/microsoft/dbt-fabric.git@main
 dbt-synapse @ git+https://github.com/microsoft/dbt-synapse.git@master
-grpcio-status~=1.47.0
```

Comment:
This was from older versions and no longer applies. It must have been left in here accidentally and didn't cause an issue until now. It causes a version conflict currently. The version that BQ installs is `1.62.0`. It was removed from the specific minor reqs files in `1.5`.